client/app/components/user/user.service.js

```diff
@@ -60,6 +60,17 @@ angular.module('services').service('User', [
       });
     };
 
+    /* eslint-disable no-unused-vars */
```

Comment:
https://github.com/ovh-ux/ovh-manager-dedicated/blob/fa6caf2ab256e3caf9863dbbad9077ac21866579/client/app/components/user/user.service.js#L52 eslint is supposed to already be disabled for that rule in this part